upgrade/prepare_for_upgrade.md

```diff
@@ -159,4 +160,7 @@ Before beginning an upgrade to a new version of CSM, there are a few things to d
       [n000]# cscli fs_info
       ```
 
+1. Optional - Create `rbd` device to provide space for the CSM release tarball.
```

Comment:
In terms of where the new procedure is linked to from the upgrade procedure: 
1) I think we may also want to link to it or mention it near the upgrade step where we pull down the tarball.  
2) Since we're going to want the tarball on ncn-m002 later, we probably should link to it from that page too, mentioning that if you copied the tarball to an rbd mount, then you can just mount that onto ncn-m002 rather than having to copy the tarball over.

Reply:
That makes sense.  and I totally forgot to put the "how to move it to the next node" procedure in there.  good catch.. no, great catch.